install_files/ansible-base/roles/build-securedrop-app-code-deb-pkg/files/changelog-xenial

```diff
@@ -1,20 +1,8 @@
-securedrop-app-code (1.8.2~rc3+xenial) xenial; urgency=medium
```

Comment:
we aren't releasing for xenial but since these files no longer exist in develop, this should not block merge